chai.js

```diff
@@ -1,6 +1,6 @@
-var chai = require('chai');
-var chaiAsPromised = require('chai-as-promised');
-var chaiFiles = require('chai-files');
+const chai = require('chai');
```

Comment:
This will need a `'use strict';` too. Even though our tests aren't covering it (unfortunately), consumers of this library running node 4 will error without it.